DuckDuckGo.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -32,8 +32,7 @@
       "kind" : "remoteSourceControl",
       "location" : "https://github.com/duckduckgo/BrowserServicesKit",
       "state" : {
-        "revision" : "dc04bd2707bd64e743961d88675606c0ea4539b1",
-        "version" : "193.0.1"
+        "revision" : "e2545b64853587b28dc850f0f3be3e621fd18a6f"
```

Comment:
This diff looks weird, shouldn't there be a `version` string in here too, like there was for 193.0.1?